src/api/app/models/event/relationship_create.rb

```diff
@@ -4,6 +4,12 @@ class RelationshipCreate < Relationship
     self.description = 'Relationship was created'
 
     receiver_roles :any_role
+
+    def originator
```

Comment:
This method could go in the parent class `Relationship`, so the duplication would not be needed.